tests/alert-distance-within-1/test.yaml

```diff
@@ -0,0 +1,11 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+checks:
+
+  # Check that we only have one alert event type in eve.
```

Comment:
this checks that there are 2 alert records, but doesn't say anything about other record types

Reply:
Ah, good point, I will update. Thanks!